include/deal.II/grid/grid_tools_cache.h

```diff
@@ -150,6 +150,17 @@ namespace GridTools
     const Mapping<dim, spacedim> &
     get_mapping() const;
 
+
+    /**
+     * Return the cached covering_rtree object, constructed using
```

Comment:
`covering_rtree` is an internal object that users don't know about. Instead of referring to it in the documentation, you probably want to describe what it does or means.